cli/command/image/build.go

```diff
@@ -503,7 +504,7 @@ type resolvedTag struct {
 // "FROM <image>" instructions to a digest reference. `translator` is a
 // function that takes a repository name and tag reference and returns a
 // trusted digest reference.
-func rewriteDockerfileFrom(ctx context.Context, dockerfile io.Reader, translator translatorFunc) (newDockerfile []byte, resolvedTags []*resolvedTag, err error) {
+func rewriteDockerfileFrom(ctx context.Context, dockerfile io.Reader, translator translatorFunc, istrusted bool) (newDockerfile []byte, resolvedTags []*resolvedTag, err error) {
```

Comment:
Looks like the whole reason this `rewriteDockerfileFrom()` file is there, is for docker content trust. Instead of passing this boolean, can we just skip the whole call to this function?

So, actually, looks like that's already what's done; it's only called in that case, so we can just remove the boolean (and check). This is the only location this function is called:
https://github.com/docker/cli/blob/82a80858857aeeb37008bd7d2f4afcda3d39c966/cli/command/image/build.go#L288-L305